cmd/kots/cli/install.go

```diff
@@ -59,6 +61,44 @@ func InstallCmd() *cobra.Command {
 
 			log := logger.NewCLILogger()
 
+			if !v.GetBool("skip-rbac-check") {
```

Comment:
This check is not needed if ensure-rbac is false: ` && v.GetBool("ensure-rbac")`